scripts/output/vue.js

```diff
@@ -2,17 +2,31 @@ import { mkdirSync, writeFileSync } from 'node:fs'
 import { join as joinPath } from 'node:path'
 import { getSVGs, getDirname } from '../util/helpers.js'
 import chalk from 'chalk'
+import defaultIconDescriptions from '../../default-icon-descriptions.js'
 
 const __dirname = getDirname(import.meta.url)
 const icons = []
 const basepath = joinPath(__dirname, '../../vue/')
 mkdirSync(basepath, { recursive: true })
 
-getSVGs().forEach(({ svg, filename, exportName }) => {
+// Create Vue Icon
+getSVGs().forEach(({ svg, filename, exportName, name }) => {
+    // Handle i18n of icon title
+    const iconNameCamelCase = exportName.replace('Icon', '').replace(/\d+/g, '');
```

Comment:
Just a suggestion to simplify but feel free to ignore this one if you don't like it.

```suggestion
    const iconNameCamelCase = exportName.replace(/Icon|\d+/g, '');
```